simplify/simplify_env/bin/Activate.ps1

```diff
@@ -0,0 +1,230 @@
+<#

```

Comment:
Usually virtualenv stuff is gitignored, there's no need to check it into source control.